catalyst/metrics/_cmc_score.py

```diff
@@ -184,4 +184,120 @@ def compute_key_value(self) -> Dict[str, float]:
         return kv_metrics
 
 
-__all__ = ["CMCMetric"]
+class ReidCMCMetric(AccumulationMetric):
```

Comment:
could you please also add it to the `catalyst/metrics/__init__.py` and the [docs](https://github.com/catalyst-team/catalyst/blob/master/docs/api/metrics.rst)?